### DISCUSSION TITLE
Can AlphaTab Player load multiple Scores?

### DISCUSSION THREAD

**Discussion by alexrln** [CoderLine/alphaTab#1100] [Q&A] [answered]
Hi! I'm not very computer-literate/skilled at coding, so apologies if this is a stupid question, but can the AlphaTab Player load multiple Scores? I have several different Guitar Pro files, each corresponding to different versions/performances of the same song, and would like for visitors to be able to select whichever version they'd like view from a play-list.

**Comment by Danielku15** [comment, marked as answer]
Depending on how I interpret your question the answer differs. But in summary: 

A single alphaTab instance on your website has one file (Score data model) loaded it uses for display and playback. One file consists of multiple tracks (instruments) and alphaTab can display only single instruments, or combine multiple ones into a single music sheet. The file in an alphaTab instance can be changed through the functions it offers. 

You can have multiple separate alphaTab instances active on a single page. 

In your example it depends how you build your application. e.g. if you have a dropdown will all files and the user changes the dropdown. you can tell alphaTab through its functions to load a new file. The old one is then dropped. 

In the [early steps of the tutorial](https://alphatab.net/docs/tutorial-web/viewport#initialize-alphatab) an initial file is provided to be loaded. The [`load`](https://alphatab.net/docs/reference/api/load/) function on the API object allows you to load a new file any time from different sources like URLs or raw data like `Uint8Array`.
